src/components/Checkbox.js

```diff
@@ -94,6 +100,8 @@ class Checkbox extends React.Component {
                 onFocus={this.onFocus}
                 onBlur={this.onBlur}
                 ref={this.props.forwardedRef}
+                onPressOut={this.onBlur}
+                onPressIn={this.onFocus}
```

Comment:
Why do we need this?

Reply:
Oh I think we don't need  `onPressIn`, only `onPressOut` is enough.

Reply:
Yep, I removed onPressIn since we handle it inside `firePressHandlerOnClick` logic.